app/task_routes.py

```diff
@@ -0,0 +1,94 @@
+from app import db
+from app.models.task import Task 
+from datetime import datetime
+from dotenv import load_dotenv
+from flask import Blueprint, jsonify, abort, make_response, request
+import requests  
+import os
+
+load_dotenv()
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try: 
+        model_id = int(model_id)
+    except:
+        abort(make_response({
+            "message": f"{cls.__name__} {model_id} is invalid, please search by task_id."}, 400))
+    task = cls.query.get(model_id)
+    if not task:
+        abort(make_response({"message":
+        f"{cls.__name__} {model_id} does not exist."}, 404))
+    return task 
```

Comment:
One thing we could do is move this helper function into a separate file for helper functions that _aren't_ routes.